src/js/controllers/chatController.js

```diff
@@ -197,17 +203,14 @@ module.exports = ViewController.extend({
     _initConversation: function() {
         var promise;
 
-        if (this.conversationInitiated) {
+        if (this.collection.length > 0 && !this.collection.at(0).isNew()) {
```

Comment:
This is the key change that fixed the first message disappearance. It relied on `this.conversationInitiated` that was set to true further down the promise chain. I'm pretty there was a race condition when we tracked an event and/or update the user and the conversation is started there too.